### PR TITLE
Update Lesson_5_The_Rust_Layer_Cake.md

### PR DESCRIPTION
**Pull request by shungo0222**
Fix some typos.

### CODE REVIEW

File: Solana_Core/en/Core_3/Lesson_5_The_Rust_Layer_Cake.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 We've said hello. It's time to learn how to handle instruction data, just like we did with client-side development.
 
-Before we get into building, I wanna go over some of the concepts we'll be using. Remember what I said about rules, abilities and interactions? I'll walk you through the abilities and rules you need to know to make a native Solana program. The word native is important here - a lot of what we'll learn here will be handled by Anchor later on. 
+Before we get into a building, I wanna go over some of the concepts we'll be using. Remember what I said about rules, abilities, and interactions? I'll walk you through the abilities and rules you need to know to make a native Solana program. The word native is important here - a lot of what we'll learn here will be handled by Anchor later on.
```
Comment (helloitsm3) [inline Solana_Core/en/Core_3/Lesson_5_The_Rust_Layer_Cake.md:3]: This change doesn't make much sense. If you want to change it, you can change it to `Before we get into building the application, I wanna go...`

File: Solana_Core/en/Core_3/Lesson_5_The_Rust_Layer_Cake.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-The reason we're learning native development is because it's important to understand how things work under the hood. Once you understand how things work at the atomic level, you'll be able to use tools like Anchor to build even more powerful programs. Think of it like a boss fight with different types of enemies - you need to learn how to counter each individual monster (and your abilities) before taking them all on at the same time.
+The reason we're learning native development is that it's important to understand how things work under the hood. Once you understand how things work at the atomic level, you'll be able to use tools like Anchor to build even more powerful programs. Think of it like a boss fight with different types of enemies - you need to learn how to counter each individual monster (and your abilities) before taking them all on at the same time.
```
Comment (helloitsm3) [inline Solana_Core/en/Core_3/Lesson_5_The_Rust_Layer_Cake.md:5]: The sentence looks fine. Don't think we should change it

**Review by helloitsm3 — COMMENTED** [review]
Do make the appropriate changes. Will merge it once it's done. Thanks

**Comment by shungo0222** [comment]
Oh sorry. I fixed.

**Review by helloitsm3 — APPROVED** [review]
LGTM

**Pull request merged**